docs/docs/ui.md

```diff
@@ -0,0 +1,52 @@
+# Chat UI
+
+The MongoDB Chatbot UI is a React.js component library that you can use to build a chatbot UI.
+
+Currently, it's focused on internal MongoDB use cases. However, we may make it more generic in the future if there is sufficient external interest.
+
+## Install
+
+Install the `mongodb-chatbot-ui` package from npm. This contains the React.js components that you can use to build a chatbot UI.
+
+```shell
+npm install mongodb-chatbot-ui
+```
+
+## The `Chatbot` Component
+
+The Chatbot component is the main component of the library. It renders the entire chatbot UI.
+You can configure it to work with your chat server API.
+
+<details>
+
+<summary> Demo GIF </summary>
+
+![Chatbot UI Demo GIF](/img/ui-demo.gif)
+
+</details>
+
+### Usage
```

Comment:
There's a new API for this in `0.1.0` - check out the latest README https://github.com/mongodb/chatbot/blob/main/chat-ui/README.md